examples/ex_screensaver.c

```diff
@@ -0,0 +1,132 @@
+/* This example displays a picture on the screen, with support for
```

Comment:
I think we don't need two examples - ex_disable_screensaver should be good enough.

Reply:
Yep, please remove this. If ex_disable_screensaver doesn't support something, add that to that example.